ios/Exponent/Versioned/Core/EXVersionManager.mm

```diff
@@ -139,7 +139,7 @@ - (void)invalidate
 
 - (NSDictionary<NSString *, NSString *> *)devMenuItemsForBridge:(id)bridge
 {
-  RCTDevSettings *devSettings = [self _moduleInstanceForBridge:bridge named:@"DevSettings"];
+  EXDevSettings *devSettings = [bridge moduleForClass:EXDevSettings.class];
```

Comment:
Do we use specifically `EXDevSettings`' API? What would be more resilient is fetching an instance of:
```suggestion
  RCTDevSettings *devSettings = [bridge moduleForClass:[self getModuleClassFromName:"DevSettings"]];
```
(That's a suggestion for all other occurrences like this.